src/pages/Contributors/index.js

```diff
@@ -17,10 +20,11 @@ export default function Contributors({ match }) {
     const [unAffiliatedOrgs, setUnAffiliatedOrgs] = useState([]);
     const [affiliatedOpen, setAffiliatedOpen] = useState(false);
     const [unaffiliatedOpen, setUnaffiliatedOpen] = useState(false);
+
     useEffect(() => {
         const fetchData = async () => {
             try {
-                const result = await axios.get('https://test-civictechindexadmin.herokuapp.com/api/organizations/');
+                const result = await axios.get(`${process.env.REACT_APP_ORGANIZATIONS_API}/api/organizations/`);
```

Comment:
you can generify this name.

REACT_APP_API_URL

Reply:
is it because you are injecting 'REACT_APP_API_URL' specifically in the Heroku environment variables? I also have to change the key on my local .env file correct?